libvirt/resource_libvirt_domain.go

```diff
@@ -861,6 +920,27 @@ func resourceLibvirtDomainRead(d *schema.ResourceData, meta interface{}) error {
 	// or it will show as changed
 	d.Set("emulator", domainDef.Devices.Emulator)
 
+	if domainDef.Devices.Graphics != nil {
+		graphics := map[string]string{}
+		graphicsInstance := domainDef.Devices.Graphics[0]
+		graphics["type"] = graphicsInstance.Type
+		// Add either the port or autoport but not both
+		if graphicsInstance.AutoPort == "yes" {
+			graphics["autoport"] = "yes"
+		} else {
+			graphics["listen_port"] = strconv.Itoa(graphicsInstance.Port)
+		}
```

Comment:
This section doesn't cover `no` for `AutoPort`.

Reply:
Thanks, that's very helpful. Been staring at it for too long.